pages/docs/data-modeling/default-exploration.mdx

```diff
@@ -0,0 +1,72 @@
+---
+description: Data models support a default exploration which is used as a starting point any time a new exploration is started from the data model.
+---
+
+# Default Model Exploration
+
+Data models support the ability to control the default exploration that is used when starting a new exploration from a data model.
+
+## Behavior without a default exploration
+
+By default, when starting a new exploration from a data model, Hashboard bootstraps a visualization using your model's measures and attributes.
+If a model has a primary date attribute, your model will be bootstrapped as a bar chart with the primary date on the x-axis and first-defined measure on the y-axis.
+If a model does not have a date attribute, your model will be bootstrapped as a table.
+
+## Configuring a default exploration
+
+Often times there may be a more useful starting point for your explorations based on a given model.
+
+To configure a default exploration for a model:
+
+1. Navigate to the Data Models page of your project.
+2. Find the model you want to configure and click the "Edit" button.
+3. Click the "Code editor" button inside the model builder.
+4. Once the code editor is open, add a new field at the root level named `defaultExploration`.
+5. Populate the `defaultExploration` field using the schemas for the `Data` and `Visualization` objects as defined in [saved explorations](/docs/data-ops/config-schema/saved-explorations). Both the `data` and `visualization` objects are required.
```

Comment:
Consider adjusting this to use the Explorer to build the config, vs having to do it by hand (at least, that's how I would do it). Something like:

1. Configure an Exploration for your model that you want to use as the default.
2. Open up the code editor and copy the entire `data` and `visualization` fields.
...
5. Add a `defaultExploration` field and paste in the `data` and `visualization` fields you copied in step 2.

Reply:
Good suggestion. I updated the instructions to start the user off in the Data Explorer.